profiles/sg/game.lua

```diff
@@ -0,0 +1,51 @@
+root.ActiveRenderer = RendererType.OpenGL;
+
+root.LayerCount = 100;
+root.GameFeatures = GameFeature.Sc3VirtualMachine | GameFeature.Renderer2D | GameFeature.Input | GameFeature.Audio |
+GameFeature.Video | GameFeature.DebugMenu;
+-- NOTE: 1080p messes up rendering of "loading system data..." message,
```

Comment:
1080p is correct for S;G HD. But you are reusing the system message box from C;H LCC which is a base 720p game. You can try to reuse the system message box for example from Memories Off 8 (which looks/acts pretty similar) and it should look correct.